patch-image.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/bash
+set -ex
+
+patch_file="/tmp/${PATCH_LIST}"
+
+while IFS= read -r line
+do
+    # each line is in the form "project_dir refs" where:
+    # - project is the last part of the project url including the org, for
+    # example openstack/ironic
+    # - refs is a list of comma-separated gerrit refs, for example
+    # refs/changes/67/759567/1,refs/changes/67/787632/11
+    PROJECT=$(echo $line | cut -d " " -f1)
+    PROJ_NAME=$(echo $PROJECT | cut -d "/" -f2)
+    PROJ_URL="https://opendev.org/$PROJECT"
+    REFSPECS=$(echo $line | cut -d " " -f2)
+
+    cd /tmp
+    git clone "$PROJ_URL"
+    cd "$PROJ_NAME"
+    for REFSPEC in "${REFSPECS//,/ }"; do
```

Comment:
It doesn't make sense to checkout in a loop. We should just provide the refspec of the last change instead of a list.